doc/classes/Object.xml

```diff
@@ -374,6 +374,7 @@
 			<argument index="1" name="value" type="Variant">
 			</argument>
 			<description>
+				Set property into the object, after the current frame's physics step. This is equivalent to calling [method set] via [method call_deferred], i.e. [code]call_deferred("set", [property, value])[/code].
```

Comment:
It took me a couple tries to understand this, but I'm not sure if that's your fault.

If I may make a suggestion (though I'm not sure how good this suggestion is):
```suggestion
				Sets a property on the object, after the current frame's physics code has finished. This is equivalent to calling [method set] via [method call_deferred], i.e. [code]call_deferred("set", [property, value])[/code].
```

Also, is it setting it after *all* nodes have had their physics code run? Or only after the current object's physics code? That might be a good distinction to make.

Reply:
Oh, nevermind. The issue got closed before I could submit my review.

Reply:
Your suggestion was how I was initially going to word this, but I noticed that `set` was worded like this. So I chose to stick to the same language as that docstring, for the sake of consistency.

I agree with you though. I think it might be easier to understand the way you suggest.